pkgs/applications/misc/razergenie/default.nix

```
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitHub, lib, meson, ninja, pkgconfig, qtbase, qttools
+, wrapQtAppsHook
+, enableExperimental ? false
+, includeMatrixDiscovery ? false
+}:
+
+let
+  version = "0.8.1";
+  pname = "razergenie";
+
+in stdenv.mkDerivation rec {
```

`rec` not needed since no attribute inside of the set is referenced within the set
```suggestion
in stdenv.mkDerivation {
```